src/test/java/com/truelayer/java/acceptance/MandatesAcceptanceTests.java

```diff
@@ -155,6 +159,121 @@ public void itShouldGetFunds() {
         assertEquals(true, getConfirmationOfFundsResponseApiResponse.getData().getConfirmed());
     }
 
+    @Test
+    @DisplayName("It should get mandate constraints")
+    @SneakyThrows
+    public void itShouldGetConstraints() {
+        // create mandate
+        ProviderSelection preselectedProvider =
+                ProviderSelection.preselected().providerId(PROVIDER_ID).build();
+        CreateMandateRequest createMandateRequest =
+                createMandateRequest(SWEEPING, preselectedProvider, Limit.PeriodAlignment.CONSENT);
+        ApiResponse<CreateMandateResponse> createMandateResponse =
+                tlClient.mandates().createMandate(createMandateRequest).get();
+        assertNotError(createMandateResponse);
+
+        // save the params to be used later
+        Integer paymentAmount = 1;
+        Integer calculatedRemainingPerMonth = createMandateRequest
+                        .getConstraints()
+                        .getPeriodicLimits()
+                        .getMonth()
+                        .getMaximumAmount()
+                - paymentAmount;
```

Comment:
I would not bother doing calculations here. from the perspective of the library the test is passed if responses are aligned with the API refs.